### ISSUE TITLE
Add Contributing.md 

### ISSUE THREAD

**Issue by IterRoboticsClub** [IterRoboticsClub/GettingStartedWith-Git-GitHub#6]
Here's a link of a good Contributing guidelines that can be used for reference
[atom/CONTRIBUTING.md](https://github.com/atom/atom/blob/master/CONTRIBUTING.md)

**Comment by SouravReal** [comment]
Yes I'm getting a feeling that i can. And sorry for replying lately.
